app/main/services/AnalyticsService.js

```diff
@@ -24,7 +24,7 @@ export default class AnalyticsService extends ServiceBase {
         try{
             this.mixpanel = Mixpanel.init('e80db0ad2789b5718fa1b84b6661f008');
         } catch(e){
-            console.log('mixpanel e', e);
+            console.error('mixpanel e', e);
```

Comment:
log.error shouldn't be used for this, since IDE can be run from internal networks with no access to internet. Therefore such error should not be displayed to the user.
log.warn is more appropriate because it doesn't display error dialog in production mode